lib/dialects/sqlite3/query/compiler.js

```diff
@@ -12,25 +12,30 @@ const {
   identity,
 } = require('lodash');
 
-function QueryCompiler_SQLite3(client, builder) {
-  QueryCompiler.call(this, client, builder);
+class QueryCompiler_SQLite3 extends QueryCompiler {
+
+  constructor(client, builder) {
+    super(client, builder);
 
-  const { returning } = this.single;
+    const { returning } = this.single;
 
-  if (returning) {
-    this.client.logger.warn(
-      '.returning() is not supported by sqlite3 and will not have any effect.'
-    );
+    if (returning) {
+      this.client.logger.warn(
+        '.returning() is not supported by sqlite3 and will not have any effect.'
+      );
+    }
+    const emptyStr = constant('');
+    this.forShare = emptyStr;
+    this.forUpdate = emptyStr;
   }
-}
 
-inherits(QueryCompiler_SQLite3, QueryCompiler);
+  // inherits(QueryCompiler_SQLite3, QueryCompiler);
```

Comment:
Why the commented out code?

Reply:
Oh, I thought I got rid of all of them. 

Just an old habit that makes it easier for me to review changes before committing.